babblesim/libc/src/strlcat.c

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
this is copied from baselibc so should have same license  (put LICENSE from core/baselibc into babblesim/libc folder)